### ISSUE TITLE
Texture Bug

### ISSUE THREAD

**Issue by Chaosleo64** [Refaltor77/CopperVanillaMods#2]
Nice idea, but the textures don't load with either the plugin or your Addfood plugin. With the CopperVanillaMods plugin, the armor has a texture when worn, but nothing more, what could be the reason for this and how do I fix this?

**Comment by Atara6431** [comment]
Have you fixed this issue? I have the same problem

**Comment by Chaosleo64** [comment]
No, unfortunately not, I have already tried to create an extra texture pack, but it didn't work either.
